modular_skyrat/modules/apc_arcing/apc.dm

```diff
@@ -1,3 +1,10 @@
+/// Lower excess value for APC arcing, 5% chance to arc
+#define APC_ARC_LOWERLIMIT 2500000
+/// Moderate excess value for APC arcing, 10% chance to arc
+#define APC_ARC_MEDIUMLIMIT 5000000
+/// Upper excess value for for APC arcing, 15% chance to arc
+#define APC_ARC_UPPERLIMIT 7500000
```

Comment:
Can be further defined or code commented to portray what the number translates to in metric.